nova3/engines/piratebay.py

```diff
@@ -70,8 +74,13 @@ def search(self, what, cat='all'):
         params = {'q': what}
         if category != '0':
             params['cat'] = category
-        response = retrieve_url(base_url % urlencode(params))
-        response_json = json.loads(response)
+
+        url = base_url % urlencode(params)
+
+        data = self.retrieve_url(url)
+
+        # response = retrieve_url(base_url % urlencode(params))
+        response_json = json.loads(data)
```

Comment:
Could you please streamline the changes? Try make the minimal change from the original.